app/src/main/res/layout/notifications_list_item.xml

```diff
@@ -9,12 +10,30 @@
         android:orientation="vertical"
         android:padding="8dp">
 
-        <TextView
-            android:id="@+id/title"
-            android:layout_width="wrap_content"
+        <LinearLayout
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:textAppearance="@style/NavView.TextView.Medium"
-            tools:text="Dzisiaj 1 to szczęśliwy numerek" />
+            android:orientation="horizontal">
+
+            <com.mikepenz.iconics.view.IconicsImageView
+                android:id="@+id/notificationIcon"
+                android:layout_width="24dp"
+                android:layout_height="24dp"
+                android:scaleType="centerInside"
+                app:iiv_color="?android:textColorSecondary"
+                app:iiv_icon="cmd-chevron-down"
+                app:iiv_size="24dp"
+                android:layout_gravity="center_vertical"
```

Comment:
Move the gravity to the LinearLayout to center both the icon and text vertically.